src/components/dialog/dialog.interactions.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
+import { StoryObj } from "@storybook/react";
+import { userEvent, within, expect, waitFor } from "@storybook/test";
+
+import Dialog from ".";
+import Button from "../button";
+import { allowInteractions } from "../../../.storybook/interaction-toggle/reduced-motion";
+import Typography from "../typography";
+import Textbox from "../textbox";
+import Form from "../form";
+
+type Story = StoryObj<typeof Dialog>;
+
+export default {
+  title: "Dialog/Interactions",
+  component: Dialog,
+  parameters: { layout: "padded" },
+  decorators: [],
+} as const;
```

Comment:
```suggestion
};
```

we don't need to coerce here